braindecode/datasets/base.py

```diff
@@ -113,7 +122,8 @@ def set_description(self, description, overwrite=False):
             self._description = pd.concat([self.description, description])
 
     def _target_name(self, target_name):
-        if target_name is not None and not isinstance(target_name, (str, tuple, list)):
+        if target_name is not None and not isinstance(target_name,
+                                                      (str, tuple, list)):
```

Comment:
```suggestion
        if target_name is not None and not isinstance(target_name, (str, tuple, list)):
```
Please do not use linter on code part you did not wrote, as it will be confusing to know code authorship (with git blame for example).
Could you revert those changes?